src/chat/components/InputToolbar.tsx

```diff
@@ -93,43 +98,41 @@ const InputToolbar: React.FC<IInputToolbar> = ({
     } catch (error) {
       console.error('Error while opening gallery:', error);
     }
-  }, [onSend]);
+  }, [libraryOptions, onSend]);
 
   return (
     <View style={[styles.container, containerStyle]}>
-      <View>
-        {renderLeftCustomView && renderLeftCustomView()}
-        {hasCamera && (
-          <PressableIcon
-            icon={cameraIcon}
-            iconStyle={flattenedIconStyle}
-            onPress={onPressCamera}
-          />
-        )}
-        {hasGallery && (
-          <PressableIcon
-            onPress={onPressGallery || openGallery}
-            icon={galleryIcon}
-            iconStyle={flattenedIconStyle}
-          />
-        )}
-        <View style={[styles.composeWrapper, composeWrapperStyle]}>
-          <ScrollView scrollEnabled={false}>
-            <Composer
-              {...props}
-              textInputStyle={[styles.textInput, composerTextInputStyle]}
-            />
-          </ScrollView>
-        </View>
-        {!!text && (
-          <PressableIcon
-            iconStyle={flattenedIconStyle}
-            onPress={() => onSend?.({ text: text }, true)}
-            icon={iconSend}
+      {renderLeftCustomView && renderLeftCustomView()}
+      {hasCamera && (
+        <PressableIcon
+          icon={cameraIcon}
+          iconStyle={flattenedIconStyle}
+          onPress={onPressCamera}
+        />
+      )}
+      {hasGallery && (
+        <PressableIcon
+          onPress={onPressGallery || openGallery}
+          icon={galleryIcon}
+          iconStyle={flattenedIconStyle}
+        />
+      )}
+      <View style={[styles.composeWrapper, composeWrapperStyle]}>
+        <ScrollView scrollEnabled={false}>
```

Comment:
why we need scrollView here?

Reply:
it is already in `master`, but I removed it